src/providers/krb5/krb5_auth.c

```diff
@@ -1023,6 +1023,9 @@ static void krb5_auth_done(struct tevent_req *subreq)
         goto done;
 
     default:
+        DEBUG(SSSDBG_IMPORTANT_INFO,
+              "The krb5_child process returned an error. Please inspect the "
+              "krb5_child.log file for more information\n");
```

Comment:
Given that the message is printed into the logs or the journal in the first place, I'm quite confident the admin will already know their location :-)